docs/source/faq.md

```diff
@@ -115,6 +115,34 @@ It allows customization of the various parameters. An environment variable
 has precedence over the value defined in `ucx.conf`.
 The file can be created using `ucx_info -Cf`.
 
+#### Build user application with UCX
+
+UCX supports metainformation subsystem based on `pkg-config` tool. To add UCX support
+into application output from `pkg-config` tool may be used in Makefile:
+```
+program: program.c
+        cc program.c $(pkg-config --cflags --libs ucx)
+```
+To build static application where UCX is used particular TL-specific modules
```

Comment:
maybe something like:
`To build static application with UCX supporting just specific network transport modules (e.g. CMA and KNEM), the following command can be used:`